src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -35,18 +35,22 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
             var stringType = semanticModel.Compilation.GetSpecialType(SpecialType.System_String);
-            if (stringType == null)
+
+            if (stringType is null)
```

Comment:
This if condition is unreachable. Only `GetTypeByMetadataName` can return null. For special type if it doesn't exist, the compiler returns an error symbol.

Reply:
I see. Looks like the initial code was wrong then. Fixing